makeflow/src/dag_file.h

```diff
@@ -26,14 +26,23 @@ typedef enum {
 	DAG_FILE_STATE_UP
 } dag_file_state_t;
 
+typedef enum {
+	DAG_FILE_TYPE_INPUT,
+	DAG_FILE_TYPE_OUTPUT,
+	DAG_FILE_TYPE_INTERMEDIATE
```

Comment:
Could you add comments to these ones? Specially because  DAG_FILE_TYPE_INTERMEDIATE could be input/output of a node, but INPUT/OUTPUT refers to the dag. Correct?